### PR TITLE
boofuzz’s web-GUI logging listen to any address

### PR DESCRIPTION
**Pull request by tin-z**
Choose which address expose boofuzz’s web-GUI logging, with 'localhost' as the default value.

### CODE REVIEW

**Review by jtpereyda — CHANGES_REQUESTED** [review]

File: boofuzz/fuzz_logger_curses.py [hunk 1]
```diff
@@ -83,6 +84,7 @@ def __init__(
 
         self._title = "boofuzz"
         self._web_port = web_port
+        self._address_listening = address_listening
```
[the remark this answers is not in the document]
Comment (tin-z) [reply boofuzz/fuzz_logger_curses.py]: Ok ;)

**Review by jtpereyda — APPROVED** [review]
Looks good! Would you mind adding a line to CHANGELOG.rst under upcoming features?

**Comment by jtpereyda** [comment]
Oh, and see also the stickler-ci finding on line 343.

**Comment by tin-z** [comment]
Ok done!

Now i see a timeout  as result in executing 'Python 3.7 on windows-latest' test

**Comment by SR4ven** [comment]
The run was just suck. It happens from time to time, rerunning it worked.
Thanks for your work @tin-z

**Comment by tin-z** [comment]
Thank you! Have a nice one

**Pull request merged**